docs/tools/tools_intro.md

```diff
@@ -19,12 +19,12 @@ The Mbed Online Compiler is our in-house IDE and should be familiar to anyone wh
 
 | Architecture version | Compiler |
 |----                  |----           |
-| Arm v6M       | `Arm Compiler 6.11` or `Arm Compiler 5.06 update 6` based on the ARM toolchain supported by the target (see `supported_toolchains` in [Adding and configuring targets](../reference/adding-and-configuring-targets.html)) |
-| Arm v7M       | `Arm Compiler 6.11` or `Arm Compiler 5.06 update 6` based on the ARM toolchain supported by the target (see `supported_toolchains` in [Adding and configuring targets](../reference/adding-and-configuring-targets.html)) |
+| Arm v6M       | `Arm Compiler 6.11` |
+| Arm v7M       | `Arm Compiler 6.11` |
 | Arm v7A       | `Arm Compiler 6.11` |
 | Arm v8M       | `Arm Compiler 6.11` |
 
-<span class="notes">**Note**: Arm Compiler 6 is the default Arm Compiler version for Mbed OS development. All platforms are already compatible with it. Please do not use Arm Compiler 5 in any new development as it will be deprecated in the future.</span>
+<span class="notes">**Note**: Mbed OS 6 does not support Arm Compiler 5 or uARM.</span>
```

Comment:
Same comment to make this line future proof.

Reply:
I'm sorry, I thought I left a question here. 

https://os.mbed.com/docs/mbed-os/development/build-tools/index.html

There is a list of supported compilers for Mbed CLI. That *seems* like a useful "single source of truth" (without the ARMC5 line, of course). So going with your suggestion, tidying up this list and making it more prominent so I can link everywhere else to it seems useful. 

Can you tie the names of the compilers in this list to their parameter names? For example, is there any use for `ARM` when now it can only mean `ARMC6`?

`ARM`, `ARMC5`, `UARM`, `ARMC6`, `GCC_ARM` and `IAR`